README.md

```diff
@@ -95,14 +98,21 @@ For example, to create and save the letsencrypt state in the reverse-proxy home:
     sudo docker run -ti --rm -p 80:80 \
         -v "/home/reverse-proxy/letsencrypt:/etc/letsencrypt" \
         -v "/home/reverse-proxy/letsencrypt:/var/lib/letsencrypt" \
-        certbot/certbot:v1.30.0 certonly
-
-And then to copy the key and certificate:
-
-    sudo cp /home/reverse-proxy/letsencrypt/live/domain.name/fullchain.pem \
-        /home/reverse-proxy/cert.pem
-    sudo cp /home/reverse-proxy/letsencrypt/live/domain.name/privkey.pem \
-        /home/reverse-proxy/key.pem
+        certbot/certbot:v1.32.2 certonly
+
+Repeat the above step for both the auth service and web service domain names.
+Then copy the keys and certificates:
+
+    export AUTH_DOMAIN=my_domain_name_hosting_the_auth_service
+    sudo cp /home/reverse-proxy/letsencrypt/live/${AUTH_DOMAIN}/fullchain.pem \
+        /home/reverse-proxy/auth-cert.pem
+    sudo cp /home/reverse-proxy/letsencrypt/live/${AUTH_DOMAIN}/privkey.pem \
+        /home/reverse-proxy/auth-key.pem
+    export WEB_DOMAIN=my_domain_hosting_the_back-end_web_service
+    sudo cp /home/reverse-proxy/letsencrypt/live/${WEB_DOMAIN}/fullchain.pem \
+        /home/reverse-proxy/auth-cert.pem
+    sudo cp /home/reverse-proxy/letsencrypt/live/${WEB_DOMAIN}/privkey.pem \
+        /home/reverse-proxy/auth-key.pem
```

Comment:
This is all tech debt to be solved with #64 or so, right? Manually running certbot and copying certificates and keys around is not very sustainable!

Reply:
Yes. For only one machine, a few minutes every three months is sustainable, but if we grow to more machines, it is not.

Reply:
Balancing the many hours of cost today to the project versus the long-term hour here or there is tricky : )

Reply:
It's not sustainable to have copy/paste errors in the things to be copied and pasted, though! Latter two should be `web-cert.pem` etc. This is part of the debt you refer to.

Reply:
> Yes. For only one machine, a few minutes every three months is sustainable, but if we grow to more machines, it is not.

This assumes that there is someone available and able to spend a few minutes every few months! (And, side note, if we wait until the very last minute we are more likely to have outages, so call it every two months.) I want the project to succeed, but in case it goes into limbo, having the site break seems likely to me to decrease its chances of coming out of limbo.

We can discuss relative priorities of work, but I would advocate for getting this automated before the end of the month.